ObjcVoiceQuickstart/ViewController.m

```diff
@@ -303,6 +303,18 @@ - (void)callInviteReceived:(TVOCallInvite *)callInvite {
 
     NSLog(@"callInviteReceived:");
     
+    NSString *callKitProviderName = @"Voice Quickstart\n";
+    if (callInvite.callerInfo.verified != nil && [callInvite.callerInfo.verified boolValue]) {
+        callKitProviderName = @"✅ Caller Verified\n";
+    }
+    
+    CXProviderConfiguration *configuration = [[CXProviderConfiguration alloc] initWithLocalizedName:callKitProviderName];
```

Comment:
This would definitely make the objc implementation look better, but unfortunately we still need to initialize the ivar in the `init` method in the Swift implementation.

Reply:
In stead of having iVar, Can we have symmetric implementation for caller side also?